puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/tfmROR52.groovy

```diff
@@ -0,0 +1,67 @@
+def packages_to_build
+
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '15'))
+        disableConcurrentBuilds()
+    }
+
+    stages {
+        stage('Clone Packaging') {
+            steps {
+
+                deleteDir()
+
+                checkout([
+                    $class : 'GitSCM',
+                    branches : [[name: '*/tfm-ror52']],
+                    userRemoteConfigs: [
+                        [url: 'https://github.com/theforeman/rails-packaging']
+                    ]
+                ])
+
+            }
+        }
+        stage('Find packages') {
+            steps {
+
+                copyArtifacts(projectName: env.JOB_NAME, optional: true)
+
+                script {
+
+                    if (fileExists('commit')) {
+                        commit = readFile(file: 'commit').trim()
+
+                        if (commit) {
+                            packages_to_build = find_changed_packages("${commit}..HEAD")
+                        }
+                    } else {
+                        packages_to_build = 'all'
+                    }
+                }
+            }
+        }
+        stage('Release Build Packages') {
+            steps {
```

Comment:
I think `find_changed_packages` can return `[]`, would this step then need a condition not to run? like we do here?

https://github.com/theforeman/foreman-infra/blob/5656e2708758b9f7761b8839d6725ea49ece841c/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/tfmROR51.groovy#L33-L35